templates/swagger/v1_json.tmpl

```diff
@@ -20339,6 +20339,9 @@
           "type": "string",
           "x-go-name": "CloneURL"
         },
+        "commit_status_state": {
+          "$ref": "#/definitions/CommitStatusState"
+        },
```

Comment:
Hmm… The repository is returned quite a few times.
In all these instances, the latest commit state is empty.
I don't think I can allow this bug to be merged.

Reply:
What should I do if I only need this for the dashboard repo list?

Reply:
That is a good question where I'm not sure of the answer myself…
In the worst case a wrapper struct?

Reply:
Hmmm... That would make this a breaking PR for that search endpoint if I create a new struct with 2 fields
```go
type DashboardRepository struct {
	Repository *Repository `json:"repository"`
	LatestCommitStatusState CommitStatusState `json:"latest_commit_status_state"`
}
```

Is there a way to create a struct by extending another? Something like
```go
type DashboardRepository & Repository struct {
	LatestCommitStatusState CommitStatusState `json:"latest_commit_status_state"`
}
```

Reply:
Not modifying the API anymore